screens/Challenges/ToDoScreen.js

```diff
@@ -25,26 +25,42 @@ export default function ToDoScreen() {
   function renderTodos() {
     let todos = [];
     let alternator = true;
+    console.log("\n\n\nEntered renderTodos")
+    // console.log(openedChallenges)
     for (challenge of openedChallenges) {
```

Comment:
Should be `let challenge of openedChallenges`

Reply:
Or actually `const`

Reply:
wait, is this why it wasn't saving...